shared/src/main/scala/ListHelpers.scala

```diff
@@ -61,10 +61,14 @@ object ListHelpers:
         return VList(to.zipWithIndex.map { (item, index) =>
           f.execute(item, index, List(item))
         }*)
+
+    val params = f.originalAST match
+      case Some(lam) => lam.params
+      case None      => List.empty
```

Comment:
```suggestion
    val params = f.originalAST.fold(Nil)(_.params)
```
I'm not sure how complex this would seem to someone new to Scala, so don't accept if you think this makes the code less understandable

Reply:
If I don't know what it does exactly, then there's a chance others won't too :p